src/panel/layouts.js

```diff
@@ -26,13 +21,20 @@ export function getMapPaddings({ isMobile, isDirectionsActive, isIframe }) {
   const topUIElement = isDirectionsActive ? '.direction-panel' : '.top_bar';
   const topUIHeight = document.querySelector(topUIElement)?.clientHeight || 0;
   return {
-    bottom: mobileBottomPanelHeight + ADDITIONAL_PADDING / 2,
+    bottom: MOBILE_BOTTOM_PADDING,
     top: topUIHeight + ADDITIONAL_PADDING / 2,
-    right: 70,
+    right: 20,
```

Comment:
I think that the reasoning behind the value 70 was to avoid meaningful items (e.g route outlines or markers) to be hidden by the floating button on the right.  
Is that not still relevant ?

Reply:
I took this shortcut so that the usable map viewport is symmetric, otherwise POIs are not visually centered on mobile.
Actually I've always thought this was too much of map space potentially wasted for a single button which poses no problem on the majority of cases…
Maybe I was too eager to simplify all this.